docs/install/index.md

```diff
@@ -430,6 +432,44 @@ You can use the Maven packages defined in the following `dependency` to include
 </div> <!-- End of scala -->
 
 
+
+<div class="java">
+<div class="gpu">
+<br/>
+You can use the Maven packages defined in the following `dependency` to include MXNet in your Java project. The Java API is provided as a subset of the Scala API and is intended for inference only. Please refer to the <a href="java_setup.html">MXNet-Java setup guide</a> for a detailed set of instructions to help you with the setup process.
+
+<a href="https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~~1.3.1-SNAPSHOT~~"><img src="https://img.shields.io/badge/org.apache.mxnet-linux gpu-green.svg" alt="maven badge"/></a>
+
+```html
+<dependency>
+    <groupId>org.apache.mxnet</groupId>
+    <artifactId>mxnet-full_2.11-linux-x86_64-gpu</artifactId>
+    <scope>system</scope>
```

Comment:
Please also add `<version></version>` here

Reply:
I used the setup guide as a reference... so it would seem that would need updating as well? http://34.201.8.176/versions/java_install/install/java_setup.html

Reply:
Yes it does. This is something we learned in the bugbash. You can update it in this pr or I'll grab it a little later.

Reply:
for this we should be able to just put:
<version>1.3.1</version>

When we release 1.4.0 these instructions will all be updated to reflect the java api being available in the official release which is on maven.